### PR TITLE
Header button

### PR DESCRIPTION
**Pull request by Fx-Colliot**
there is no .eslintcache !!! \o/

### CODE REVIEW

**Review by Thoms-C — DISMISSED** [review]
Seems good to me !

File: src/components/HeaderBody.jsx [hunk 1]
```diff
@@ -21,14 +27,16 @@ export default function HeaderBody() {
...
+        <AuthorSentence> la barre très haut" Agnès Varda</AuthorSentence>
       </ContainerAuthorSentence>
       <>
-        <ContainerMoreInformation>
-          <MoreInformationSentence>Plus d'informations</MoreInformationSentence>
+        <ContainerMoreInformation hidden={hidden} onClick={onClick}>
+          <MoreInformationSentence>En savoir plus</MoreInformationSentence>
         </ContainerMoreInformation>
+        {hidden ? <Main /> : null}
+        {hidden ? <Footer /> : null}
```
Comment (LoicBrassart) [inline src/components/HeaderBody.jsx]: Tip: you can replace with
```
{hidden || {<Main/><Footer />}}
```

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by Thoms-C — APPROVED** [review]
Value changed ok !

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**